build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+set -x
+TAG=`date +%Y-%m-%d`
```

Comment:
I've seen style guides prefer $() to backticks. I'm not really sure what the benefits are, but we should probably be consistent.